_includes/about.html

```diff
@@ -0,0 +1,14 @@
+<p>CivicActions Accessibility is an open project primarily developed by CivicActions team members.</p>
+
+<p>We follow [agile process](/agile) and encourage others to do so as well.</p>
+
+<p>We are gathering information that we find useful for our team, and hope that it resonates with our clients and the broader community. We acknolwedge that this resource is imperfect, impermanent and incomplete, and we encourage you to help make it better. There is a lot of great information about digital accessibility already freely available on the web. We aim to refer people to great resources, or build upon them.</p>
+
+<p>Please [let us know](https://accessibility.civicactions.com/contact) what you find helpful, if there are resources we should include or if there are ways we can improve.</p>
```

Comment:
@jaube05 because this is HTML the markdown link will need to be modified to be an HTML link on this line and line 3 above